testdata/fuzz/invariant/InvariantReentrancy.t.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: Unlicense
```

Comment:
Totally insane, automatic reentrancy detection lol.

I think there might be a bug here in how the trace is generated. shouldn't the `hello` -> `backdoor` call be as a subtree of the previous one? i.e. smth like below

```
  [7469] Vulnerable::hello()
    ├─ [854] Vulnerable::hello()
    │   ├─ [98] Malicious::world()
    │    │      └─ ← [513] Vulnerable::backdoor()
    |    |                       └─ ← ()
    │   └─ ← ()
    └─ ← ()
```

maybe i misunderstand the trace

<img width="1326" alt="image" src="https://user-images.githubusercontent.com/17802178/182056766-da1945de-9927-41da-8ba2-f3e6c3a4616a.png">

Reply:
yeah, that's the main reason I mentioned it needs some more work. Actually, we never enter the overridden functions (in this case world), so there's no trace of it.